### ISSUE TITLE
[toolkit] Error building when interceptor folder is missing

### ISSUE THREAD

**Issue by Rajith90** [wso2/product-microgateway#715]
### Description:
Following error occurs when a project without interceptor folder is build.
`org.wso2.apimgt.gateway.cli.exception.CLIInternalException: Error occurred while generating source code for the open API definitions.
	at org.wso2.apimgt.gateway.cli.cmd.BuildCmd.execute(BuildCmd.java:120)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.wso2.apimgt.gateway.cli.cmd.Main.main(Main.java:48)
Caused by: java.nio.file.NoSuchFileException: /var/jenkins_home/workspace/MicroGatewayPipeline/MGW-Projects/bookstore/interceptors
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixCopyFile.copy(UnixCopyFile.java:526)
	at sun.nio.fs.UnixFileSystemProvider.copy(UnixFileSystemProvider.java:253)
	at java.nio.file.Files.copy(Files.java:1274)
	at org.wso2.apimgt.gateway.cli.utils.GatewayCmdUtils.copyFolder(GatewayCmdUtils.java:686)
	at org.wso2.apimgt.gateway.cli.utils.GatewayCmdUtils.copyAndReplaceFolder(GatewayCmdUtils.java:635)
	at org.wso2.apimgt.gateway.cli.cmd.BuildCmd.execute(BuildCmd.java:108)
	... 2 more
`

### Steps to reproduce:
This happens when we check out project from git hub any other VCS.Because interceptor folder is empty, those folder will not be committed. Hence upon checkout and building it gives the error

### Affected Product Version:
3.0.1

### OS, DB, other environment details and versions:
- OS:
- Client:
- DB:
- Env (Docker/K8s):

---
#### Related Issues:
https://github.com/wso2/product-microgateway/issues/714

**Comment by praminda** [comment]
Planning to add `.keep` file to interceptors directory.

**Comment by praminda** [comment]
Fixes are added to all version specific branches. Issue will be autoclosed once jballerina branch is merged into master

**Comment by praminda** [comment]
All fixes are merged. Closing the issue to flush the open issue list
